### ISSUE TITLE
NYU Handoff

### ISSUE THREAD

**Issue by saulshanabrook** [jupyterlab/jupyterlab-data-explorer#113]
All of these items should be completed before our demo to NYU

### Ready for Review:
- [x] Binder link works 
- [x] Dev docs up to date
- [x] User facing docs include user stories and instructions https://github.com/jupyterlab/jupyterlab-data-explorer/issues/86
- [x] GitHub issues and milestones for current and future work are up to date
  - [x] Issue and PR templates.
  - [x] Appropriate labels exist and are applied to issues/PRs.
  - [x] Milestone for current and next release exists.
  - [x] Issues and PRs are targeted to appropriate Milestone.
- [x] Test Suite up to date
  - [x] Setup testing infrastructure for local and CI based tests.
  - [x] Documented on how to run test suite locally.

**Comment by tgeorgeux** [comment]
### Reviewed:
- [ ] Binder link works 
- [ ] Dev docs up to date
- [ ] User facing docs include user stories and instructions https://github.com/jupyterlab/jupyterlab-data-explorer/issues/86
- [ ] GitHub issues and milestones for current and future work are up to date
  - [ ] Issue and PR templates.
  - [ ] Appropriate labels exist and are applied to issues/PRs.
  - [ ] Milestone for current and next release exists.
  - [ ] Issues and PRs are targeted to appropriate Milestone.
- [ ] Test Suite up to date
  - [ ] Setup testing infrastructure for local and CI based tests.
  - [ ] Documented on how to run test suite locally.
Please make a comment below to note which of the items you've reviewed as you check items off the list.
